examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>Same for time_range.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Some comment that I did before.